lib/src/visitor/async_evaluate.dart

```diff
@@ -141,6 +141,9 @@ class _EvaluateVisitor
   /// All modules that have been loaded and evaluated so far.
   final _modules = <Uri, Module>{};
 
+  /// Configuration seen by a module URI.
```

Comment:
"seen by" is unclear. Maybe "used to load"? It's probably also worth clarifying that these are the *first* configurations used to load each module.